### PR TITLE
chore: Added scss and misc cleanup of code style and structure in vue components

### PR DESCRIPTION
**Pull request by magnuh**

### CODE REVIEW

File: docs/.vitepress/Tabs.vue [hunk 1]
```diff
@@ -1,44 +1,41 @@
...
-  };
+import { ref } from 'vue';
 
+const props = defineProps({
+  initialTab: String,
+  tabs: Array
+});
+const activeTab = ref(props.initialTab);
+const switchTab = (tab) => {
+  activeTab.value = tab;
```
Comment (AnnaRybkina) [inline docs/.vitepress/Tabs.vue:10]: ✅

File: docs/.vitepress/Tabs.vue [hunk 2]
```diff
@@ -1,44 +1,41 @@
...
+};
 </script>
 
-<style scoped> 
+<template>
+<div class="tab">
```
Comment (AnnaRybkina) [inline docs/.vitepress/Tabs.vue]: Just curious, no indentation inside `<template>` for `<div>` ?

**Review by AnnaRybkina — APPROVED** [review]
Great stuff 👍 Tnx for cleaning the mess with old syntax

File: docs/.vitepress/Tabs.vue (hunk 2 above)
Comment (magnuh) [reply docs/.vitepress/Tabs.vue]: There should be in the template yes... I fix. 👍

**Pull request merged**